cg/meta/workflow/nf_analysis.py

```diff
@@ -235,6 +300,57 @@ def write_trailblazer_config(self, case_id: str, tower_id: str) -> None:
             file_path=config_path,
         )
 
+    def create_sample_sheet(
+        self,
+        case_id: str,
+        dry_run: bool,
+    ):
+        """Create sample sheet for a case."""
+        sample_sheet_content: list[list[Any]] = self.get_sample_sheet_content(case_id=case_id)
+        if not dry_run:
+            self.write_sample_sheet(
+                content=sample_sheet_content,
+                file_path=self.get_sample_sheet_path(case_id=case_id),
+                header=self.sample_sheet_headers,
+            )
+
+    def create_params_file(
+        self,
+        case_id: str,
+        dry_run: bool,
+    ):
+        """Create parameters file for a case."""
+        LOG.debug("Getting parameters information")
+        workflow_parameters: dict | None = self.get_workflow_parameters(case_id=case_id).dict()
+        if self._append_params_to_nextflow_config:
+            workflow_parameters = None
+        if not dry_run:
+            self.write_params_file(case_id=case_id, workflow_parameters=workflow_parameters)
+
+    def create_nextflow_config(self, case_id: str, dry_run: bool = False) -> None:
+        """Create nextflow config file."""
+        if content := self.get_nextflow_config_content(case_id=case_id):
+            LOG.debug("Writing nextflow config file")
+            if dry_run:
+                return
+            write_txt(
+                content=content,
+                file_path=self.get_nextflow_config_path(case_id=case_id),
+            )
+
+    def config_case(
+        self,
+        case_id: str,
+        dry_run: bool,
+    ):
+        """Create directory and config files required by a workflow for a case."""
+        self.create_case_directory(case_id=case_id, dry_run=dry_run)
+        self.create_sample_sheet(case_id=case_id, dry_run=dry_run)
+        self.create_params_file(case_id=case_id, dry_run=dry_run)
+        self.create_nextflow_config(case_id=case_id, dry_run=dry_run)
+        if dry_run:
+            LOG.info("Dry run: Config files will not be written")
```

Comment:
Wouldn't it make sense to log this before all the other writing logs ("Writing nextflow config file", "Writing..)

```suggestion
        if dry_run:
            LOG.info("Dry run: Config files will not be written")
        self.create_case_directory(case_id=case_id, dry_run=dry_run)
        self.create_sample_sheet(case_id=case_id, dry_run=dry_run)
        self.create_params_file(case_id=case_id, dry_run=dry_run)
        self.create_nextflow_config(case_id=case_id, dry_run=dry_run)
```